pages/download/index.html

```diff
@@ -37,12 +37,12 @@ <h1>Try the latest</h1>
 <p>Help us make Silverblue better by testing the <b>Rawhide</b> builds of Silverblue that
 will eventually become Fedora 31.</p>
 
-<p><a class="button" href='https://download.fedoraproject.org/pub/fedora/linux/development/rawhide/Silverblue/x86_64/iso/Fedora-Silverblue-ostree-x86_64-Rawhide-20190428.n.0.iso'><svg class="icon"><use href="/public/svg-sprites.svg#download" /></svg>64-bit 2.1GB Silverblue image (Fedora Rawhide)</a></p>
+<p><a class="button" href='https://www.mirrorservice.org/sites/dl.fedoraproject.org/pub/fedora/linux/development/rawhide/Silverblue/x86_64/iso/Fedora-Silverblue-ostree-x86_64-Rawhide-20190504.n.0.iso'><svg class="icon"><use href="/public/svg-sprites.svg#download" /></svg>64-bit 2.1GB Silverblue image (Fedora Rawhide)</a></p>
```

Comment:
I'd rather not see us rely on an external service for pointing at the ISO; let's just keep `download.fedoraproject.org` and update the ISO used:

https://download.fedoraproject.org/pub/fedora/linux/development/rawhide/Silverblue/x86_64/iso/Fedora-Silverblue-ostree-x86_64-Rawhide-20190501.n.0.iso

Reply:
That will be https://dl.fedoraproject.org/pub/fedora/linux/development/rawhide/Silverblue/x86_64/iso/ and same for iso